pcsx2/Patch_Memory.cpp

```diff
@@ -394,6 +394,8 @@ void _ApplyPatch(IniPatch *p)
 	case CPU_EE:
 		switch (p->type)
 		{
+		u64 mem;
+		u64 ledata;
```

Comment:
Can you move these two to the beginning of the _ApplyPatch() function and initialise them to 0 please.
I don't know how declaring them in a switch like this is valid, it shouldn't be lol.

Reply:
Alright, will do, sorry about all these switch issues lol I don't usually use them. I think the compiler didn't complain because I put the variables within the brackets of the switch case

Reply:
Addressed in https://github.com/PCSX2/pcsx2/pull/4255/commits/94b198085c3679ae3cf55c45cc4bb9bf20493fd0